Investment strategy (#305)
niteoweb/handbook: 2_Operations/investment-strategy.md

# Investment Strategy

> Niteo keeps reserves to [cover close to 12 months of average expenses](/5_People/profit-sharing.md). To not lose this money to inflation, we invest it into appreciating assets.

Reviewer: ```suggestion
At Niteo, we like to be anti-fragile, so we can focus on the long term. We want to be able to survive an economic crisis or our projects suddenly spiralling out. For this reason, we keeps monetary reserves to [cover close to 12 months of average expenses](/5_People/profit-sharing.md).

In order to not lose this considerable chunk of money to inflation, we invest it into appreciating assets.
```

## Short-Term Pool

> Reserves in safe and liquid assets. We try to keep the currency equally split between EUR and USD. Around 30% of total reserves.

Reviewer: ```suggestion
Reserves in safe and liquid assets. We try to keep the currency equally split between EUR and USD to not have all our eggs in one basket. Minimum 30% of all reserves should be in this pool so that we have cash on hand if we need to act quick.
```

Assets:

> - cash on bank accounts

Reviewer: ```suggestion
- cash on bank accounts (minimum 15% of total, default reserve)
```

> - bonds (15% of total) with Vanguard BNDW (total world bonds in USD)

Reviewer: ```suggestion
- bonds (15% of total) with [Vanguard BNDW](https://investor.vanguard.com/etf/profile/overview/bndw) (World-wide investment-grade bonds, in USD)
```

## Long-Term Pool

> Reserves in more risky or illiquid assets. The currency split is not relevant because most of these assets compensate for the difference between EUR and USD. Around 70% of total reserves.

Reviewer: ```suggestion
Reserves in more risky or illiquid assets. The currency split is irrelevant because most of these assets are global investments. Up to 70% of all reserves can be in this pool.
```

> - real estate (20% of total) with [Niteo House](https://house.niteo.co) (EUR)

Reviewer: ```suggestion
- real estate (mas 20% of total) with [Niteo House](https://house.niteo.co) (EUR)
```

> - stock ETFs (max 30% of total) with Vanguard VWRA (global stocks in USD)

Reviewer: ```suggestion
- stock ETFs (max 30% of total) with [Vanguard VWRA](https://www.vanguardinvestments.dk/portal/instl/dk/en/product.html#/fundDetail/etf/portId=9679/assetCode=equity/?overview) (global stocks, in USD)
```

> - private funds (max 30% of total) with Calm Company Fund, Ben Tossel's No-Code Fund, etc.

Reviewer: ```suggestion
- private funds (max 30% of total) with [Calm Company Fund](https://calmfund.com/), [Ben Tossel's No-Code Fund](https://angel.co/v/back/ben-tossell), etc.
```

- loans (max 10% of total)
- FOMO (max 5%) -> crypto stuff, individual stocks

## Summary

> Keep in mind the above percentages are maximum for each asset class and are not supposed to be all maxed out (that's why the total comes out to 110%). For example: if we have maxed out real estate, stock ETFs and private funds, we're already over the recommended maximum for long-term pool.

Reviewer: ```suggestion
Keep in mind the above percentages are maximum for each asset class and are not supposed to be all maxed out (that's why the total comes out to 110%). For example: if we have maxed out real estate, stock ETFs and private funds, we're already over the recommended maximum for long-term pool, and need to refrain from buying more crypto.
```
